src/server.rs

```diff
@@ -262,8 +262,16 @@ impl Server {
     /// Adds a new response header to our concurrent hashmap
     /// this can be called after the server has started.
     pub fn add_response_header(&self, key: &str, value: &str) {
-        self.global_response_headers
-            .insert(key.to_string(), value.to_string());
+        match key.to_lowercase().as_str() {
```

Comment:
Hey @18o 👋 

Thank you for the PR 😄 TIL about the web headers fact. Thank you ✨ 

However, I have one question. Why not make everything lowercase?

Reply:
It's a good question, I was only thought about the `content-type` thing.
Do you mean all keys in the `global_response_headers` make them into lowercase?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

> An HTTP header consists of its case-insensitive name followed by a colon (:), then by its value. [Whitespace](https://developer.mozilla.org/en-US/docs/Glossary/Whitespace) before the value is ignored.

I read this at MDN docs today.

Reply:
@18o , 
> Do you mean all keys in the global_response_headers make them into lowercase?

Reply:
Yeah, we can make the keys in `global_response_headers ` all uppercase or lowercase, in order to avoid issues like this.

Reply:
@18o , sounds good. Let us make everything `lower_case` . I haven't seen someone mistype all upper case headers.

Reply:
ok, it's implemented in the latest commit